dataland-frontend/src/components/messages/FailMessage.vue

```diff
@@ -3,6 +3,7 @@
     <Message severity="error" :sticky="true" :key="messageId" :closable="closable">
       <h4>Sorry, but an Error occurred.</h4>
       <p>{{ message }}</p>
+      <p v-if="summary">{{ summary }}</p>
```

Comment:
What is this summary?

Reply:
Because we send more error information as a summary:
![image](https://github.com/d-fine/Dataland/assets/96523011/47164808-8ac7-495e-8483-ddf87e0d3e1c)

Reply:
yes